Content.Shared/Power/EntitySystems/SharedPowerReceiverSystem.cs

```diff
@@ -1,6 +1,18 @@
-﻿namespace Content.Shared.Power.EntitySystems;
+using Content.Shared.Examine;
+using Content.Shared.Power.Components;
+
+namespace Content.Shared.Power.EntitySystems;
 
 public abstract class SharedPowerReceiverSystem : EntitySystem
 {
-
+    ///<summary>
+    ///Adds some markup to the examine text of whatever object is using this component to tell you if it's powered or not, even if it doesn't have an icon state to do this for you.
+    ///</summary>
+    protected void OnExamined(EntityUid uid, SharedApcPowerReceiverComponent component, ExaminedEvent args)
```

Comment:
Use `Entity<SharedAPcPowerReceiverComponent>` please

Reply:
These are the arguments that are required by `SubscribeLocalEvent`. I don't think there's an overload accepting `Entity<T>`, is there?

Reply:
There is.

Reply:
why did you copy paste it into server and client systems thats what shared is for
`private void OnExamined(Entity<SharedApcPowerReceiverComponent> ent, ref ExaminedEvent args)` in shared and nothing in server or client